### ISSUE TITLE
An estimate of when the  tf TextVectorization layer will be implemented in tfjs? 

### ISSUE THREAD

**Issue by zliendo** [tensorflow/tfjs#3303]
I have a question

#### TensorFlow.js version
1.7.4r1
#### Browser version
Google Chrome Version 81.0.4044.138 (Official Build) (64-bit)
#### Describe the problem or feature request
Is there a timeline to implement the tf TextVectorization layer in tfjs? 

I build a keras model in python using this layer:
from tensorflow.python.keras.layers.preprocessing import text_vectorization
TextVectorization.  
it works, code snippet:
```
   vectorize_layer = text_vectorization.TextVectorization(
          max_tokens=400,
          output_mode='tf-idf', name="textvectorizing")
    vectorize_layer.adapt(np.array(text_data))
    input_text = tf.keras.Input(shape=(1,),dtype=tf.string, name="text")
    text_vectorized = vectorize_layer(input_text)
``` 
but when I imported to tfjs, I get this message

```
errors.ts:48 Uncaught (in promise) Error: Unknown layer: TextVectorization. This may be due to one of the following reasons:
1. The layer is defined in Python, in which case it needs to be ported to TensorFlow.js or your JavaScript code.
2. The custom layer is defined in JavaScript, but is not registered properly with tf.serialization.registerClass().
    at new t (errors.ts:48)
    at deserializeKerasObject (generic_utils.ts:242)
    at deserialize (serialization.ts:31)

```

Thank you!

**Comment by rthadur** [comment]
@zliendo we will not be implementing this op anytime in near future. Do you want to contribute ?
Thank you

**Comment by zliendo** [comment]
Thank you @rthadur for the prompt response, 
ok, I will give it a try on contributing its implementation,
I'm building a POC, for the final model I may use embeddings instead.

I'll read about how to contribute

**Comment by zliendo** [comment]
Thank you!

**Comment by zliendo** [comment]
Hi @imartinezl ,
I did implement js logic to do the tf-idf transform vectorization (which works for the POC that I am working on), but I haven't implemented it as a layer for a tensorflow model yet. I did get approval from my director to contribute to this open source repository, still I have to go through my company levels of approval for this type of external contributions (routine reviews), then it may take some time before my contribution to this repository (which I am looking forward too! exciting! )
